roles/jd-server/Cargo.toml

```diff
@@ -35,4 +34,8 @@ key-utils = { version = "^1.0.0", path = "../../utils/key-utils" }
 secp256k1 = { version = "0.28.2", default-features = false, features =["alloc","rand","rand-std"] }
 siphasher = "1"
 hex = "0.4.3"
-
+base64 = "0.21.5"
+hyper = { version = "1.1.0", features = ["full"] }
+hyper-util = { version = "0.1", features = ["full"] }
+http-body-util = "0.1"
+bytes = "1"
```

Comment:
we usally use utils/buffer here double check if another dependency is really needed

Reply:
I don't understand, can you explain a bit more?

Reply:
are u sure that instead of bytes you can not use the lib crate under utils/buffer?

Reply:
I will open a PR for that later